crates/astria-composer/local.env.example

```diff
@@ -8,7 +8,7 @@ ASTRIA_COMPOSER_API_LISTEN_ADDR="0.0.0.0:0"
 ASTRIA_COMPOSER_SEQUENCER_URL="http://127.0.0.1:26657"
 
 # Execution chain ID that we want to connect to
-ASTRIA_COMPOSER_CHAIN_ID="912559"
+ASTRIA_COMPOSER_CHAIN_ID="astria"
```

Comment:
is our EVM named astria now? haha

Reply:
It needed to be a string instead of numerical network I'd, used for the sequencer not the RPC. 

The rollup helm charts actually default to "astriachain"